flink-core/src/main/java/org/apache/flink/configuration/RestOptions.java

```diff
@@ -160,6 +160,20 @@
                     .withDescription(
                             "The number of threads for the asynchronous processing of requests.");
 
+    @Documentation.Section(Documentation.Sections.EXPERT_REST)
+    public static final ConfigOption<Integer> JOBMANAGER_FUTURE_THREADS =
+            key("jobmanager.future-executor.numThreads")
+                    .defaultValue(0)
+                    .withDescription(
+                            "The number of jobmanager future-executor threads for shared services.");
+
+    @Documentation.Section(Documentation.Sections.EXPERT_REST)
```

Comment:
I think we should change this to `EXPERT_CLUSTER` if we put the options in the `ClusterOptions` class.